.github/workflows/ci.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         run: coverage report
       - name: Create XML report for codacy
         run: coverage xml
+        if: github.ref == 'refs/heads/master'
```

Comment:
@holtgrewe , does this look right to you?

Reply:
Ah, this has to be used at the point where the report is posted, which is probably below that, extend line 44/45.

Reply:
I don't get it. Can you maybe add a code review?

Reply:
OK, added the patch (quite nice with the web IDE of Github)